docsrc/news.rst

```diff
@@ -3,6 +3,15 @@
 News
 ====
 
+GeoNetwork at FOSS4G-EU 2025
+------------------------------------------------
+
+Date: 20 July 2025
+
+* `geonetwork-ui to sublime your opendata platform <https://talks.osgeo.org/foss4g-europe-2025/talk/RSYJTJ/>`__
```

Comment:
@fgravin the conference https://talks.osgeo.org/foss4g-europe-2025/talk/RSYJTJ/ page does not link to a PDF or google slides. Is your presentation available online to link to?

Reply:
found the link, this is fixed